examples/data/viewport-features.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Viewport features</title>
+
+  <link rel="stylesheet" type="text/css" href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css">
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-style/v2.3/airship.css">
+
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    #map {
+      width: 100vw;
+      height: 100vh;
+    }
+  </style>
+</head>
+
+<body>
+  <section id="map"></section>
+
+  <div class="as-map-panels">
+    <div class="as-panel as-panel--top as-panel--right as-panel--vertical">
+      <div class="as-panel__element">
+        <section class="as-box">
+          <p class="as-title">Viewport features</p>
+          <p class="as-body">
+            Get the visible features on every viewport change
+          </p>
+          <p id="result" class="as-subheader as-color--success"></p>
```

Comment:
I found difficult to get the features info with just the console.log, so I decided to bring in an airship panel. If we're ok with such a UI, I can try to update all the examples, and we can then follow the 'template'.